src/appsettings/settings.py

```diff
@@ -872,3 +881,74 @@ def transform(self, path):
         for obj in objects:
             current_object = getattr(current_object, obj)
         return current_object
+
+
+# Nested settings -------------------------------------------------------------
+class NestedSetting(DictSetting):
+    """Nested setting."""
+
+    def __init__(self, settings, name='', default=lambda: dict(),
```

Comment:
As told by Codacy, the `lambda: dict()` is not necessary here, you can directly use `dict` which is callable.

Reply:
Ah! You're too quick for me! You fixed them anyway? Good :rofl:

Reply:
Well, yes :sweat_smile:.